xbmc/input/joysticks/JoystickMonitor.cpp

```diff
@@ -22,6 +22,8 @@
 #include "Application.h"
 #include "input/InputManager.h"
 
+#define AXIS_THRESHOLD       0.75f // Axis must exceed this value to be mapped
```

Comment:
This is taken from ./xbmc/input/joysticks/generic/ButtonMapping.cpp,  makes it sense to define in header to have global?